niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/InstallPermissionTester.java

```diff
@@ -486,10 +498,19 @@ public void onAvailable(Network network) {
         // new install permissions for Q
         mPermissionTasks.put(REQUEST_PASSWORD_COMPLEXITY,
                 new PermissionTest(false, Build.VERSION_CODES.Q,
-                        () -> mDevicePolicyManager.getPasswordComplexity()));
+                        () -> {
+                            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+                                mDevicePolicyManager.getPasswordComplexity();
+                            }
+                }));
 
         mPermissionTasks.put(USE_FULL_SCREEN_INTENT,
                 new PermissionTest(false, Build.VERSION_CODES.Q, () -> {
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S_V2) {
```

Comment:
Nit: you can instantiate the `PermissionTest` with a maxSdkVersion of `Build.VERSION_CODES.S` to have the test only run on devices up through Android S.

Reply:
fixed

Reply:
Change looks good; thanks!